packages/autoconf_archive.rb

```diff
@@ -0,0 +1,20 @@
+require 'package'
+
+class Autoconf_archive < Package
+  version '2017-03-21'
+  source_url 'https://ftp.wrz.de/pub/gnu/autoconf-archive/autoconf-archive-2017.03.21.tar.xz'
```

Comment:
I think it's better to use public url `http://ftpmirror.gnu.org/autoconf-archive/autoconf-archive-2017.03.21.tar.xz` as described in https://www.gnu.org/software/autoconf-archive/Downloads.html.  This ftpmirror.gnu.org redirects us to the nearest mirror ftp site.